db/migrate/20210923133920_add_admin_to_users.rb

```diff
@@ -0,0 +1,9 @@
+class AddAdminToUsers < ActiveRecord::Migration[6.1]
+  def self.up
+    add_column :users, :admin, :boolean, :default => false
```

Comment:
good but also add to ruby code that new users will set admin false
+ write tests.